XamlControlsGallery/XamlControlsGallery.csproj

```diff
@@ -1105,19 +1117,19 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="ColorCode.UWP">
-      <Version>2.0.5</Version>
```

Comment:
This change shouldn't rev the versions for unrelated packages. Please revert these changes.